styles/planet-minecraft/catppuccin.user.css

```diff
@@ -347,21 +367,6 @@
             color: @text;
```

Comment:
There are more lines above this line here (starting at line 340) that are `.content-actions` related.